### PR TITLE
Wheels for Windows (64bit)

### PR DESCRIPTION
**Pull request by ax3l**
Build wheels for Windows users.

On a side node: Need to disable manylinux PyPy builds again due to: https://github.com/pypa/manylinux/issues/899

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -27,30 +27,37 @@ jobs:
...
-        python -m pip install cibuildwheel==1.5.5
+        python -m pip install --upgrade pip setuptools wheel
+        python -m pip install cibuildwheel==1.7.1
 
     - name: Build wheel
       env:
-        # (1,2) Skip building for Python 2.7 on all platforms
-        # (3) CMake/Pybind11 scripts not clever enough to build 32bit Windows with 64bit compiler
-        # (4) GNU 7.3.1 & MPark.Variant 1.4.0 on i686:
+        # (1,2,3) Skip building for Python 2.7 & 3.9 on all platforms
```
Comment (ax3l) [inline .github/workflows/build.yml:35]: CPython 3.5 will be dropped and 3.9 added in the next release.

**Comment by AngelFP** [comment]
Hi @ax3l, this is great! I've seen the wheels for Windows are in PyPI already. Thanks a lot!

**Review by AngelFP — COMMENTED** [review]
Installed wheels from PyPI, everything running well from my side!

**Comment by ax3l** [comment]
Thank you for testing, @AngelFP! This is great!

**Pull request merged**